### ISSUE TITLE
Add a new feature - to change the texture for explosion objects

### ISSUE THREAD

**Issue by Shahin3d** [TheGameCreators/GameGuruRepo#383]
Hello
If it chose for each object a separate explosion texture ... I think we will have a beautiful effect.
If can add texture selection in the properties section.
Right now, we have a type of explosive texture for objects.
If you add this new feature - we will build a more advanced game - Of course with different explosions.
Reminder Issues #319.

**Comment by michaeloldman** [comment]
Yes Shahin3d I agree this really is something that would improve the quality and individuality of any game.

However the response to the issue #319 that I posted on this subject some time ago was almost nonexistent so it seams you and I are alone feel this sort of thing would enhance our game design.

But for the record +100

Reliquia....

**Comment by HierToo** [comment]
I would like to see it too, but let´s be realistic. There are other features which have a higher priority to the majority of the community like AI, lightning system, standalone export and performance.

**Comment by Teabone3** [comment]
oh trust me you guys are not alone.

**Comment by GraPhiX-Guru** [comment]
I also wanted this :) so I made it happen with a liitle help from my friends lol https://forum.game-guru.com/thread/220232?page=2#msg2609133

**Comment by ErrantAI** [comment]
Changing textures is nice and all but the system itself is lackluster.  There needs to be more control over the shape, volume, dynamics, debris, etc. of explosions as a proper particle effect and not just the same sprite animation over and over.
